bodhi/server/templates/updates.html

```diff
@@ -71,6 +71,9 @@ <h3 class="font-weight-bold m-0 d-flex align-items-center">Updates
                                 "user",
                                 "updateid",
                                 "alias"]
+              macroCatReleases = [('all-current', 'all current'),
```

Comment:
Not sure about the `all-` prefix, if think if you choose in a filter `pending` it is obvious that you are going to get all the pending releases.

What do you think ?

Reply:
You're right, I'll change that

Reply:
The more I think about that, the less sure I am about. The filter `?releases=all-current,F15` is more readable than `?releases=current,F15`, where `current` seems a name of a release.

I'm thinking about adopting RH Bugzilla style and change the macro filters to `__current__`, `__pending__` and `__archived__`, which should also automatically prevent that one of those names conflicts with a true release name (for example, if Bodhi is used outside Fedora and a release is named 'current').

What do you think?

Reply:
Sounds good to me :+1: